### PR TITLE
Updated data ingestions

### LINKED ISSUE #20: Ingest institution data
#Description
Ingest institution data into AWS storage

## Acceptance Criteria
- Decision records for data storage and ingestion of CSV
- Ticket created for terraform of glue, s3, and CDN
- Glue code checked in to repo
- CSV institution data available for front end application

## Definition of Done
Pipeline status validated
[Definition of Done](https://github.com/coforma/swift-tech-challenge/blob/main/documentation/product/definition_of_done.md)

### LINKED ISSUE #70: No student population data
**Describe the bug**
The student population data is not coming though in the data mapping

** Current behavior **
The front end is hardcoding a 0 into the field

**Expected behavior**
The front end would have this data and be able to display it

**Screenshots**
image
image

## Definition of Done
[Definition of Done](https://github.com/coforma/swift-tech-challenge/blob/main/documentation/product/definition_of_done.md)

### PR DESCRIPTION
**Pull request by TheDanMiller**
# Updated data ingestion

# Summary
Updates the data ingestion flow to use AI for generative content
Closes #70 
Closes #20 

## Type of change
 New feature (non-breaking change which adds functionality)

# How To Test
1. Drop a file into S3 for instutions -- watch it process. Recommend using one that is only a line or two to verify, otherwise long waits.
2. Drop a file into S3 for applications -- watch it process. Recommend using one that is only a line or two to verify, otherwise long waits.
3. Verify in Dynamo

# Checklist:

- [x] My code follows the style guidelines of this project and has no linting errors
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, including any applicable ADRs
- [x] My changes generate no new warnings
- [ ] I have added tests that fail without these changes
- [ ] New and existing tests (unit, integration, accessibility) pass locally
- [x] Documentation updated
- [x] If there are security concerns they are addressed or ticketed after being discussed

### CODE REVIEW

File: utilities/data-ingestion/get-institution-image-from-bedrock.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            logger.info("image file does not exist in S3")
+            return False
+        else:
+            # Handle Any other type of error
+            raise
+
+
+def lambda_handler(event, context):
+    # load the json to a string
+    resp = json.loads(json.loads(json.dumps(event))["Records"][0]["body"])
```
[the remark this answers is not in the document]
Comment (TheDanMiller) [reply utilities/data-ingestion/get-institution-image-from-bedrock.py:31]: Marked as won't fix in code climate. Duplication intentional

File: utilities/data-ingestion/get-institution-description-from-bedrock.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+logger = logging.getLogger()
+bedrock_runtime = boto3.client("bedrock-runtime", "us-east-1")
+dynamodb = boto3.resource("dynamodb")
+table = dynamodb.Table("institutions")
+
+
+def lambda_handler(event, context):
+    # load the json to a string
+    resp = json.loads(json.loads(json.dumps(event))["Records"][0]["body"])
```
[the remark this answers is not in the document]
Comment (TheDanMiller) [reply utilities/data-ingestion/get-institution-description-from-bedrock.py:13]: Marked as won't fix in code climate. Duplication intentional

**Review by ntsummers1 — DISMISSED** [review]
I think you can remove that app/styles/uswds-theme. That lives in src/app/styles/uswds-theme now

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -15,18 +15,19 @@ repos:
...
     - id: check-merge-conflict
-    - id: check-yaml
     - id: trailing-whitespace
+    - id: check-yaml
     - id: end-of-file-fixer
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v4.0.0-alpha.8
+    hooks:
+      - id: prettier
+        files: '\.[jt]sx?$' # *.js, *.jsx, *.ts, *.tsx
```
Comment (gmrabian) [inline .pre-commit-config.yaml:30]: how does this change on every PR for file ingestion?

**Review by gmrabian — APPROVED** [review]

**Comment by TheDanMiller** [comment]
> I think you can remove that app/styles/uswds-theme. That lives in src/app/styles/uswds-theme now

@ntsummers1 done, can you re-review

**Review by ntsummers1 — APPROVED** [review]
Looks good to me! Thanks for removing that :D

**Pull request merged**